public_html/wp-content/themes/pattern-directory/header.php

```diff
@@ -24,60 +24,3 @@
 ?>
 <div id="page" class="site">
 	<div id="content" class="site-content">
-		<header id="masthead" class="site-header <?php echo is_home() ? 'home' : ''; ?>" role="banner">
-			<div class="site-branding">
-				<?php if ( is_home() ) : ?>
-					<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php echo esc_html_x( 'Pattern Directory', 'Site title', 'wporg-patterns' ); ?></a></h1>
-
-					<p class="site-description"><?php esc_html_e( 'Add a beautifully designed, ready to go layout to any WordPress site with a simple copy/paste.', 'wporg-patterns' ); ?></p>
-				<?php else : ?>
-					<div>
-						<a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home">
-							<?php esc_html_e( 'All Patterns', 'wporg-patterns' ); ?>
-						</a>
-						<span class="sep">/</span>
-						<span class="is-current-page">
-							<?php
-							if ( is_singular( POST_TYPE ) ) {
-								esc_html_e( 'Pattern Details', 'wporg-patterns' );
-							} else if ( is_singular() ) {
-								the_title();
-							} else {
-								the_archive_title();
-							}
-							?>
-						</span>
-					</div>
-
-					<nav id="site-navigation" class="main-navigation" role="navigation">
-						<button
-							class="menu-toggle dashicons dashicons-arrow-down-alt2"
-							aria-controls="primary-menu"
-							aria-expanded="false"
-							aria-label="<?php esc_attr_e( 'Primary Menu', 'wporg-patterns' ); ?>"
-						>
-						</button>
-
-						<div id="primary-menu" class="menu">
-							<?php
-							wp_nav_menu( array(
-								'theme_location' => 'primary',
-								'menu_id'        => 'primary-menu',
-							) );
-							?>
-						</div>
-					</nav><!-- #site-navigation -->
-				<?php endif; ?>
-			</div><!-- .site-branding -->
-		</header><!-- #masthead -->
-
-		<?php if ( ! is_singular() ) : ?>
-		<div class="notice notice-warning notice-alt" style="max-width:960px; margin:1rem auto;">
-			<p style="font-size:0.9375rem; margin:0.75rem 0;">
-			<?php
-				esc_html_e( 'You’re a bit early to the party! This directory hasn’t yet fully launched.', 'wporg-patterns' );
```

Comment:
I had this in the PHP on purpose, so that if something happens to break the JS loading again, it would still appear.

Reply:
I understand. It won't be in the right place if we do go with this approach and keep it in php.